### ISSUE TITLE
Removing Product Links

### ISSUE THREAD

**Issue by maelamb** [ApothecaryGarden/mtollwc#16]
@oakwoodgates Hi Reuben! We need to remove all product links from the Maia Toll website. Is there a simple way to search for all links to products?

**Comment by oakwoodgates** [comment]
Can you give me an example of one of these links?

**Comment by maelamb** [comment]
Sure! There is one in [this blogpost](http://maiatoll.com/personal-bloopers-battle-perfectionism/) that links to Cordyceps tincture. Most but not all would link to Herbiary, right @earthwitch? Maybe the easiest thing to do is a search of all posts in WC using "http://herbiary.com/products/" as the search term...

**Comment by oakwoodgates** [comment]
@earthwitch didn't we use some kind of link checker plugin before in order to find all the broken links on the site? Do you remember what that was?

**Comment by earthwitch** [comment]
we did! but I tried the old link and it didn't work .... Are you saying you think it was a plug-in? If it is, we can research that. I thought it was something in the backend that I couldn't find.

**Comment by oakwoodgates** [comment]
@earthwitch Yep! it was a plugin...we deactivated and deleted it I believe, but if you can remember what it was called...because I sure can't :) A clue to the name of the plugin might be in the one of those link parameters in the old link you found.
